src/client/driver/driver.js

```diff
@@ -124,6 +126,19 @@ export default class Driver {
         return this.contextStorage.getItem(TEST_SPEED);
     }
 
+    get consoleMessages () {
+        return this.contextStorage.getItem(CONSOLE_MESSAGES) || {
+            log:   [],
+            info:  [],
+            error: [],
+            warn:  []
+        };
+    }
+
+    set consoleMessages (messages) {
+        return this.contextStorage.setItem(CONSOLE_MESSAGES, messages);
```

Comment:
Ehmm, we can run out of storage quite quickly with this approach

Reply:
Thought about this.. Now the max size of the session storage in browsers is 5-10MB, so it looks not so crusial since we reset the session storage for each test run. But I agree, it would be better to keep the messages on the server.